src/metabase/util/honeysql_extensions.clj

```diff
@@ -21,6 +21,23 @@
   (intern 'honeysql.format 'quote-fns
           (assoc quote-fns :h2 (comp s/upper-case ansi-quote-fn))))
 
+(defn- str-insert
+  "Insert c in string s at index i."
+  [s c i]
+  (str c (subs s 0 i) c (subs s i)))
+
+(defn- crate-column-identifier
+  [^CharSequence s]
+  (let [idx (s/index-of s "[")]
+    (if (nil? idx)
+    (str \" s \")
+    (str-insert s "\"" idx))))
```

Comment:
https://github.com/bbatsov/clojure-style-guide#body-indentation

@mikethebeer, I highly recommend taking a moment to familiarize yourself with Clojure indentation rules and reading [this section in the Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide#source-code-layout--organization) so we can save some time going back on forth on style issues.